src/components/Explore/SingleLocationChart.tsx

```diff
@@ -113,10 +151,14 @@ const SingleLocationChart: React.FC<{
   marginRight = 20,
   barOpacity,
   barOpacityHover,
+  dateRange,
+  dataMeasure,
 }) => {
-  const dateFrom = new Date('2020-03-01');
-  const today = new Date();
-  const dateTo = today;
+  const [from, to] = dateRange;
+  const dateFrom = from;
+  // const today = new Date(); // do we want today, or computed 'to'?
+  const dateTo = to;
```

Comment:
Probably computed in `Explore.tsx` (since that component manages the state of which period is selected) and passed down as `dateRange`, that way the components inside `Explore` (single and multi chart) are consistent.